### ISSUE TITLE
Install global game binary

### ISSUE THREAD

**Issue by TheJJ** [SFTtech/openage#709]
Currently, we don't install a game binary to e.g. `/usr/bin/openage`.
The only thing that complicates this is that we launch it from python, and installing some hacky bash script sucks.
Gentoo solves this with [`python-exec`](https://wiki.gentoo.org/wiki/Project:Python/python-exec), but we have to provide something similar for other distros.

The easiest way will probably be to just install the `run.py` script or the cythonized `run` executable.

The problem that needs to be solved is how we install that binary to the "correct" path without `distutils`, as we use `CMake` instead of `distutils` for compiling and installing the code.

In a nutshell:

* [x] Extend the buildsystem to figure out the distutils-equivalent install path (we can already do that for the openage python module itself)
* [x] modify `make install` to install the `run.py` as openage launcher script

**Comment by TheJJ** [comment]
Fixed by #723
